app-drawer/app-drawer.html

```diff
@@ -246,6 +266,8 @@
         // Only transition the drawer shortly after it is attached (e.g. app-drawer-layout
         // may need to set the initial opened state which should not be transitioned).
         this.$.contentContainer.style.transitionDuration = '0s';
+
```

Comment:
Does it make sense to replace this with `this._setTransitionDuration('0s')` here?